### ISSUE TITLE
No issue, just offering Slovak translation, if You wish

### ISSUE THREAD

**Issue by dufi2profor** [pachi81/GlucoDataAuto#5]

**Comment by pachi81** [comment]
Hi,

that would be great! 
But it is an ongoing job, as there are a lot of new strings in every version.
At the moment I try using ChatGPT for new strings which works really good for my surprise. So there is less work for new strings, only checking it.

**Comment by dufi2profor** [comment]
As You wish

**Comment by pachi81** [comment]
Here is a link to the current ressources, but it includes the strings for GlucoDataHandler and GlucoDataAuto, as it uses the same code.
https://github.com/pachi81/GlucoDataHandler/blob/develop_gda/common/src/main/res/values/strings.xml

**Comment by dufi2profor** [comment]
So shoul i make copy and edit it?

**Comment by pachi81** [comment]
Yes, for the first time, just download the file, edit it and send the result to me.
I will add it and the next times, you can change it direct here in the pull request.

**Comment by pachi81** [comment]
@dufi2profor Hi,
I can create a translation using Gemini AI.
But I will need you for checking. Is it ok for you? Are you also using GlucoDataHandler?

**Comment by javiercravioto** [comment]
I can also help with an spanish translation if needed.

**Comment by pachi81** [comment]
Hi @javiercravioto, thank you! There is already a spanish translation, but of'course, you can check, if you like.

I will close this issue, as there was no further message.
